Minor fixes to map function doc (#4856)
brimdata/zed: docs/language/functions/map.md

### Description

The _map_ function applies function `f` to every element in array or set `v` and
returns an array or set of the results. Function `f` must be a function that takes

> only one argument. A [user-defined function](../statements.md#func-statements) can be used for `f`.

Reviewer: ```suggestion
only one argument. `f` may be a [user-defined function](../statements.md#func-statements).
```
